app/templates/src/components/menu/index.jsx

```diff
@@ -1,242 +1,43 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Layout, Menu } from 'antd';
 import { HomeOutlined, ClusterOutlined} from '@ant-design/icons';
 import { useHistory } from 'react-router-dom';
 import { RouteNames } from '../../utils/constants';
 import { useTranslation } from 'react-i18next';
+import {getNodes} from '../../redux/nodes/nodes.action';
```

Comment:
{**suggestion**}{**improvement**} any way we can define these as kind of global namespace and import it then to avoid the directory traversal?